src/quo/components/wallet/summary_info/view.cljs

```diff
@@ -56,6 +59,11 @@
        [network-amount
         {:network :arbitrum
          :amount  (str (:amount arbitrum) " " (or (:token-symbol arbitrum) "ARB"))
+         :theme   theme}])
+     (when show-base?
+       [network-amount
+        {:network :base
+         :amount  (str (:amount base) " " (or (:token-symbol base) "ETH"))
          :theme   theme}])]))
```

Comment:
The `divider?` prop seems to be added manually for the chains above and is very easy to miss. Would be better to generalise the logic of when to show the divider instead of manually adding `show-base?` to all the previous components' divider condition.